README.md

```diff
@@ -49,6 +49,15 @@ This addon sets `CYPRESS_baseUrl` to DDEV's primary URL in the `docker-compose.c
 
 To display the Cypress screen and browser output, you must configure a `DISPLAY` environment variable.
 
+#### Linux
+
+You may need to set up access control for the X server for this to work. Install the xhost package (one is available for all distros) and run:
+
+```sh
+export DISPLAY=:0.0
+xhost +local:docker
```

Comment:
I think `xhost +` should do everything, enabling permissions everywhere. Are you sure you need `xhost +local:docker` ?  Please try this out:

```suggestion
xhost +
```

Reply:
Running `xhost +` after setting `DISPLAY` was **not** necessary in my case, under Linux.
I did run `xhost` (without `+`) though. I will verify if this is required in the coming days.

Reply:
@bserem Looking forward to your feedback so we can continue moving forward.